lib/rspecq/reporter.rb

```diff
@@ -116,16 +116,18 @@ def humanize_duration(seconds)
     def flaky_jobs_to_sentry(jobs, build_duration)
       return if jobs.empty?
 
-      Raven.capture_message("Flaky jobs detected", level: "warning", extra: {
-        build: @build_id,
-        build_timeout: @timeout,
-        queue: @queue.inspect,
-        object: self.inspect,
-        pid: Process.pid,
-        flaky_jobs: jobs,
-        flaky_jobs_count: jobs.count,
-        build_duration: build_duration
-      })
+      jobs.each do |job|
+        filename = job.sub(/\[.+\]/, '')
+        Raven.capture_message("Flaky test file: #{filename}", level: "warning", extra: {
+          build: @build_id,
+          build_timeout: @timeout,
+          queue: @queue.inspect,
+          object: self.inspect,
+          pid: Process.pid,
+          path: job,
```

Comment:
Maybe something like `job` is a more descriptive name, instead of `path`?

Reply:
Or "flaky_job", even better.

Reply:
Essentially it is the path though

Reply:
How about `flaky_job_path` then? It should be clear to users what this is about. `path` in this context is too vague.

Reply:
Flaky is kind of redundant. What about `job_path`

Reply:
8a23191

Reply:
8a23191

Reply:
My only concern is that it's clear to the user where they can find the actual test that failed. That's why I suggested `flaky_test` (and to differentiate it from the other fields which are kinda irrelevant - object, pid etc.) But it's your call.